packages/example_hardhat/scripts/deploy.ts

```diff
@@ -0,0 +1,28 @@
+import 'hardhat';
+import {ethers} from 'hardhat';
```

Comment:
Why do we even need this file if deploy scripts are going inside `packages/deploy/` ?

Reply:
This came from the hardhat template project (when you run hardhat on an empty directory), we don't really need it.
I will remove it

Reply:
Could be good to have this as an example in `packages/deploy` so it's not wasted work :)